apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,22 @@
+#include "TutorialServo.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees, float minPulsewidthInMs,
+                             float maxPulsewidthInMs)
+    : m_servoPwmOut(servoPin),
+      m_servoRangeInDegrees(servoRangeInDegrees),
+      m_minPulsewidthInMs(minPulsewidthInMs),
+      m_maxPulsewidthInMs(maxPulsewidthInMs) {}
+
+void TutorialServo::setPositionInDegrees(const float degrees) {
+  m_servoPwmOut.pulsewidth(1 + degrees / m_servoRangeInDegrees);
```

Comment:
What if degrees was negative or greater than the servo's range? What if the min and max pulse widths were set to something other than 1ms and 2ms?